### DISCUSSION TITLE
A recent animation I did in ToonSquid for part of a music video

### DISCUSSION THREAD

**Discussion by wandrinsheep** [keiwando/toonsquid#542] [Show and tell]
[this](https://youtu.be/N98wkZ6UBYQ?si=7wan5Xi8znfbwOXt) was done in ToonSquid almost entirely. Love the powerful toolset. Scene management and symbols were very handy and the ability to tween my animation accelerated a lot of my workflow. There are some things that can be improved such as brushes and some minor bugs I ran into while working but all in all this is my favorite animation app on the iPad.

**Comment by AFC26** [comment]
How long is your video

**Reply by wandrinsheep** [reply]
28 seconds
